src/5e-SRD-Monsters.json

```diff
@@ -36716,6 +36909,7 @@
     "subtype": "human",
     "alignment": "neutral good",
     "armor_class": 10,
+    "armor_type": "10 in humanoid form, 11 (natural armor) in bear and hybrid form",
```

Comment:
This isn't an armor type? Though is a bit more confusing.

Reply:
The whole were* are confusing and barely possible to write in our structure.